src/averageAges.js

```diff
@@ -37,7 +39,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const rightWomen = people.filter(person =>
```

Comment:
maybe womenWithChild will be better variable name? same with 'rightMen' and 'rightMothers'